index.bs

```diff
@@ -356,19 +360,27 @@ even slightly or slowly, without affecting your driving direction.
 
 The orientation vector of the [=Orientation Sensor=], can be calculated the following way:
 
-The gravity vector points towards the earth's core when mostly stationary and as long as the
-device is not in free fall, there is enough vector length to project the magnetic field vector
-onto the ground plane.
+As the [=Accelerometer=] is an <a>inertial-frame sensor</a>, the gravity vector will point
+towards the sky when device is mostly stationary and as long as the device is not in free fall,
```

Comment:
hmm something must be wrong, somewhere in my text, because a cross product with something pointing north and upwards, would point towards west and not east right. Could you check to see if this is correctly described